docs/core/entity/camera.md

```diff
@@ -91,6 +91,10 @@ class MyCamera(Camera):
         """Handle the WebRTC offer and return an answer."""
 ```
 
+### RTSP to WebRTC
+
+An integration may provide a WebRTC stream for any RTSP camera using `async_register_rtsp_to_web_rtc_provider`. The current best practice is for an integration to provide the actual stream manipulation with an Add-on or external service.
```

Comment:
Maybe an example code block?

Reply:
OK, i was going to keep it sparse since this is rarely used, but at the same time callables are kind of hard to follow, so I will. Let me update https://github.com/home-assistant/core/pull/59660 to use the new API then i'll add the example code block once it looks good.

Reply:
Updated with example.

Reply:
Thanks 👍